src/ServiceControl.Monitoring.Transports.RabbitMQ/ConnectionFactory.cs

```diff
@@ -3,15 +3,14 @@
     using System;
     using System.Net.Security;
     using System.Security.Authentication;
-    using System.Security.Cryptography.X509Certificates;
-    using global::RabbitMQ.Client;
+   using global::RabbitMQ.Client;
 
     class ConnectionFactory
     {
         readonly global::RabbitMQ.Client.ConnectionFactory connectionFactory;
         readonly object lockObject = new object();
 
-        public ConnectionFactory(ConnectionConfiguration connectionConfiguration, X509CertificateCollection clientCertificates, bool disableRemoteCertificateValidation, bool useExternalAuthMechanism)
+        public ConnectionFactory(ConnectionConfiguration connectionConfiguration, bool disableRemoteCertificateValidation, bool useExternalAuthMechanism)
```

Comment:
I know the original intention of this class was to be an exact copy of the one from the transport, so that's why the `clientCertificates` parameter was there.  Perhaps it would be better to leave it as-is?

Reply:
Ah, I didn't know that. What was the rationale for that?  Easier copy/paste should the transport change?

Reply:
Yes, that was the primary reason I believe.

Reply:
Ok, I've reverted the code now